database/index.js

```diff
@@ -6,7 +6,7 @@ const pool = new Pool({
 });
 
 module.exports = {
-  getAll: (callback) => {
+  getOne: (callback) => {
```

Comment:
This is named getOne now, changed from getAll, but it still gets all by id.

Reply:
GOT IT.